test/src/dash-adapter.spec.js

```diff
@@ -1414,7 +1414,7 @@ describe('DashAdapter: request filter', () => {
     dashInstance.load();
   });
 
-  it('should apply promise filter for manifest', done => {
+  it.skip('should apply promise filter for manifest', done => {
```

Comment:
can we just stub fetch or xhr themselves instead of shaka's?
I see it as we have a couple of options - 
just get the entire fetch plugin, inherit from it and wrap the call to fetch
set spy on fetch and check the call url.

Reply:
@OrenMe I tried to stub fetch and xhr and it doesn't work.. the stub callback doesn't call on window.fetch/xhr.
I'll try the inherit option

Reply:
Maybe override the function instead of stub?

Reply:
https://stackoverflow.com/questions/45946284/mock-http-fetch-in-sinon
does that help maybe?

Reply:
@RoyBregman override which function?

Reply:
Oh, you wrote that they're using their own implementation... Thought the stubbing didn't work as expected...

Reply:
https://github.com/google/shaka-player/blob/606457488f77e9cf5cac2c82e94bb4ce35d50e2a/test/net/http_plugin_unit.js
So maybe this helps?

Reply:
it seems that we can use the parse api call on those plugins. It is public and documented here for usage: https://github.com/google/shaka-player/blob/master/docs/tutorials/upgrade-v2-5.md#built-in-network-scheme-plugin-changes

Reply:
The stub on parse doesn't work since it does not exist in compile mode.

Reply:
ok, let's remove for now and open a task to rethink how to test correctly.

Reply:
@OrenMe solved